dashboards/sidekiq/main.yml

```diff
@@ -2,6 +2,10 @@ metric_keys:
   - sidekiq_job_count
 dashboard:
   title: Sidekiq
+  description: |
```

Comment:
shouldn't this be one level up? Or do we want users to customize this for their own use? Where would  we show this?

Reply:
We'll show this in the "dashboard detected" email to give context about what we detected and what you can do with it. We can also show it on the dashboard page itself to provide context about the dashboard. 

If we expose it to customers, they can provide context too, for example: "Dashboard for `<thing>`, graph `x` and `y` are important for `<reason>`. If `z` happens, do `<stuff>`"

Reply:
Sounds good! Even if we have no place to show it now, we can add something for it later